### ISSUE TITLE
Investigate dataset packing for fine-tuning efficiency

### ISSUE THREAD

**Issue by niznik-dev** [niznik-dev/cruijff_kit#378]
## Context

Split from #323 (compute efficiency meta-issue).

torchtune supports dataset packing (`packed=true`), which combines multiple short sequences into a single training sample to reduce padding waste. We already use `packed=true` as a default for ACS tasks, but there's more to investigate:

- **Packing ratio estimation**: How much speedup does packing actually give for our typical datasets? We currently don't guess packing ratios (per training defaults).
- **Interaction with max_seq_len**: Packing is most beneficial when sequences are much shorter than max_seq_len. Worth profiling for different dataset/model combos.
- **Validation with packing**: Does packing affect validation loss reporting? Need to verify torchtune handles this correctly.

## Acceptance Criteria

- [ ] Profile training time with and without packing on a representative dataset
- [ ] Document when packing helps vs. when it's negligible
- [ ] Ensure packing is correctly wired through design-experiment → scaffold-experiment

**Comment by niznik-dev** [comment]
<img src="https://raw.githubusercontent.com/niznik-dev/llm-personas/main/fusions/mattie/mattie-icon.png" width="36" align="absmiddle"> 🎻 **MxC** ✳️

Closing as part of the project archive. Correction to the body: packing was never actually enabled. `--packed` defaults to `False` and nothing in the pipeline sets it, so runs carry some padding inefficiency. Recorded in the handoff notes for whoever wants to validate packing with our custom datasets.

—MxC
